nginx-sidecar/handlers/healthcheck_handler.go

```diff
@@ -0,0 +1,43 @@
+package handlers
+
+import (
+	"log"
+	"net/http"
+	"net/url"
+)
+
+func HealthcheckHandler(w http.ResponseWriter, r *http.Request) {
+	urls := r.URL.Query()["url"]
+
+	if len(urls) < 1 {
+		log.Println("missing parameter url")
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	for _, urlToCheck := range urls {
+		urlToCheck, err := url.Parse(urlToCheck)
+
+		if err != nil {
+			log.Printf("url format error: %q", err)
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		resp, err := http.Get(urlToCheck.String())
+
+		if err != nil {
+			log.Printf("healthcheck request error: %q", err)
```

Comment:
What do you think about also sending the error message on the response body instead of only logging it? It could make debugging potential problems easier.

Reply:
Yeah, this could be useful. I'll add it to the response body.